ocs_ci/helpers/dr_helpers.py

```diff
@@ -549,3 +561,127 @@ def get_all_drpolicy():
     drpolicy_obj = ocp.OCP(kind=constants.DRPOLICY)
     drpolicy_list = drpolicy_obj.get(all_namespaces=True).get("items")
     return drpolicy_list
+
+
+def enable_fence(drcluster_name):
+    """
+    Once the managed cluster is fenced, all communication
+    from applications to the ODF external storage cluster will fail
+
+    Args:
+        drcluster_name (str): Name of the DRcluster which needs to be fenced
+    """
+
+    logger.info(
+        f"Edit the DRCluster resource for {drcluster_name} cluster on the Hub cluster"
+    )
+    restore_index = config.cur_index
+    config.switch_acm_ctx()
+    fence_params = f'{{"spec":{{"clusterFence":"{constants.ACTION_FENCE}"}}}}'
+    drcluster_obj = ocp.OCP(resource_name=drcluster_name, kind=constants.DRCLUSTER)
+    assert drcluster_obj.patch(
+        params=fence_params, format_type="merge"
+    ), f"Failed to patch {constants.DRCLUSTER}: {drcluster_name}"
+    logger.info(f"Successfully fenced {constants.DRCLUSTER}: {drcluster_name}")
+    config.switch_ctx(restore_index)
+
+
+def enable_unfence(drcluster_name):
+    """
+    The OpenShift cluster to be Unfenced is the one where applications
+     are not currently running and the cluster that was Fenced earlier.
+
+    Args:
+        drcluster_name (str): Name of the DRcluster which needs to be fenced
+    """
+
+    logger.info(
+        f"Edit the DRCluster resource for {drcluster_name} cluster on the Hub cluster"
+    )
+    restore_index = config.cur_index
+    config.switch_acm_ctx()
+    unfence_params = f'{{"spec":{{"clusterFence":"{constants.AcTION_UNFENCE}"}}}}'
+    drcluster_obj = ocp.OCP(resource_name=drcluster_name, kind=constants.DRCLUSTER)
+    assert drcluster_obj.patch(
+        params=unfence_params, format_type="merge"
+    ), f"Failed to patch {constants.DRCLUSTER}: {drcluster_name}"
+    logger.info(f"Successfully unfenced {constants.DRCLUSTER}: {drcluster_name}")
+    config.switch_ctx(restore_index)
```

Comment:
It also seems to me that it would make sense to create 3rd function which would handle setting of the expected clusterFence state, while fence/unfence functions would just used it and specified expected values (via `constants.ACTION_UNFENCE` and `ACTION_FENCE`).

Reply:
done, created separate function as `fence_state`